demo/init.sh

```diff
@@ -474,24 +473,26 @@ elif [ -n "$url_document" ]; then
 
     # (2) upload the document to api-server via the `/v1/files` endpoint
     printf "    * Uploading the document to LlamaEdge API Server ...\n\n"
-    doc_response=$(curl -X POST http://127.0.0.1:8080/v1/files -F "file=@$doc_filename")
+    doc_response=$(curl -X POST http://127.0.0.1:$llamaedge_port/v1/files -F "file=@$doc_filename")
     id=$(echo "$doc_response" | grep -o '"id":"[^"]*"' | cut -d':' -f2 | tr -d '"')
     filename=$(echo "$doc_response" | grep -o '"filename":"[^"]*"' | cut -d':' -f2 | tr -d '"')
     rm $doc_filename
     printf "\n"
 
     # (3) chunk the document
     printf "    * Chunking the document ...\n\n"
-    chunk_response=$(curl -X POST http://127.0.0.1:8080/v1/chunks -H "accept: application/json" -H "Content-Type: application/json" -d "{\"id\":\"$id\",\"filename\":\"$filename\"}")
-    chunks=$(echo "$chunk_response" | jq -c '.chunks')
+    chunk_response=$(curl -X POST http://127.0.0.1:$llamaedge_port/v1/chunks -H "accept: application/json" -H "Content-Type: application/json" -d "{\"id\":\"$id\",\"filename\":\"$filename\"}")
+
+    chunks=$(echo $chunk_response | grep -o '"chunks":\[[^]]*\]' | sed 's/"chunks"://')
 
     printf "\n"
 
     # (4) compute the embeddings for the chunks and upload them to the Qdrant instance
     printf "    * Computing the embeddings and uploading them to the Qdrant instance ...\n\n"
 
-    data=$(jq -n --arg model "all-MiniLM-L6-v2-ggml-model-f16" --argjson input "$chunks" '{"model": $model, "input": $input}')
-    embedding_response=$(curl -X POST http://127.0.0.1:8080/v1/embeddings -H "accept: application/json" -H "Content-Type: application/json" -d "$data")
+    data={\"model\":\"$embedding_model_stem\",\"input\":"$chunks"}
+
+    embedding_response=$(curl -X POST http://127.0.0.1:$llamaedge_port/v1/embeddings -H "accept: application/json" -H "Content-Type: application/json" -d "$data")
```

Comment:
The embeddings are created here. But we are missing another step to add the embeddings + source into a vector collection?